notebooks/slsim_single_lens_simulation.ipynb

```diff
@@ -0,0 +1,339 @@
+{
```

Comment:
<p>same here. Perhaps just provide some default values that you state come from DP0 data products</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LSST-strong-lensing/slsim/pull/313/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>